### DISCUSSION TITLE
Congrats ! App Developers.

### DISCUSSION THREAD

**Discussion by ghost** [ish-app/ish#1295] [General]
Congrats on being on the First Position on the App Store !
![8BF55E9B-33C5-432E-A9FB-8F8813A2A435](https://user-images.githubusercontent.com/74538195/108602623-eeae8400-73c8-11eb-9975-d81ea734bf41.png)

**Comment by 62f** [comment]
Where's the link to issue where this's been referenced? 

__Originally posted by @tbodt in https://github.com/ish-app/ish/issues/1296#issuecomment-782783571__
